Mathlib/Algebra/Homology/DerivedCategory/Ext/EnoughInjectives.lean

```diff
@@ -0,0 +1,127 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Homology.DerivedCategory.Ext.ExactSequences
+
+/-!
+# Smallness of Ext-groups from the existence of enough injectives
+
+Let `C : Type u` be an abelian category (`Category.{v} C`) that has enough injectives.
+If `C` is locally `w`-small, i.e. the type of morphisms in `C` are `Small.{w}`,
+then we show that the condition `HasExt.{w}` holds, which means that for `X` and `Y` in `C`,
+and `n : ℕ`, we may define `Ext X Y n : Type w`. In particular, this holds for `w = v`.
+
+However, the main lemma `hasExt_of_enoughInjectives` is not made an instance:
+for a given category `C`, there may be different reasonable choices for the universe `w`,
+and if we have two `HasExt.{w₁}` and `HasExt.{w₂}` instances, we would have
+to specify the universe explicitly almost everywhere, which would be an inconvenience.
+Then, we must be very selective regarding `HasExt` instances.
```

Comment:
Can you add this to the docstring of `hasExt_of_enoughInjectives` as well? Thanks.